Translated/Tasks/Side/Hoxie/ss0030761201.csv

```diff
@@ -1,3 +1,3 @@
 Title#0,"""Aerio Subjugation Exercise III"""
-Summary#0,"""The gatekeeper of Central City's North Gate, Hoxie, recommends you defeat enemies at Resol Forest to gain more field experience."""
+Summary#0,"""The gatekeeper of Central City's Northern Gate, Hoxie, recommends you defeat enemies at Resol Forest to gain more field experience."""
```

Comment:
Having "Northern" here makes the punctuation fall off the end. There's no graceful solution that's perfectly consistent across all four, but this is the least inconsistent, so just leave this one how it is.